### PR TITLE
Add "lyrics" as a command

### PR DESCRIPTION
**Pull request by Arsukeey**
I've added the feature to show the lyrics for the song that is currently playing.
However, it adds a dependency: lyricwikia.

Sorry if you don't want more dependencies to your project, but I thought it would be cool to add this feature, because I downloaded your program for it, and I was surprised when I didn't find it.

### CODE REVIEW

**Comment by pwittchen** [comment]
Hi @Arsukeey.

Thanks for your PR and contribution. It's nice idea and we can add such functionality. Nevertheless, I don't like a few things in the code. I've added my suggestions in the code review. Moreover, please fix code formatting (you can find details in the `README.md`) because CI is failing and squash your commits into one.

Regards,
Piotr

**Comment by Arsukeey** [comment]
Thank you. I was changing some variable names for testing and I forgot to remove/rename that.

**Comment by pwittchen** [comment]
Thanks for the update.

There's still wrong formatting, not updated `setup.py` file and multiple commits. You can use rebaseandsqua.sh website for that.

**Comment by Arsukeey** [comment]
Thanks for the website!
I updated the setup.py and the code now complies with pep8.

**Comment by pwittchen** [comment]
Thanks,

Can you squash your 8 commits into one? I prefer to have more organized history on git (1 commit / PR). It can be done by rebasing and squashing the commits.

**Comment by Arsukeey** [comment]
Done! :-D

File: spotifycli/spotifycli.py [hunk 1]
```diff
@@ -85,19 +88,20 @@ def get_arguments():
         ("--version", "shows version number"),
         ("--status", "shows song name and artist"),
         ("--statusshort", "shows status in a short way"),
-        ("--song", "shows the song name"),
-        ("--songshort", "shows the song name in a short way"),
+        ("--song", "shows the title name"),
```
Comment (pwittchen) [inline spotifycli/spotifycli.py]: Can you change it back to `song`?

File: spotifycli/spotifycli.py [hunk 2]
```diff
@@ -85,19 +88,20 @@ def get_arguments():
...
+        ("--songshort", "shows the title name in a short way"),
```
Comment (pwittchen) [inline spotifycli/spotifycli.py]: Can you change it back to `song`?

File: spotifycli/spotifycli.py [hunk 3]
```diff
@@ -85,19 +88,20 @@ def get_arguments():
...
         ("--playbackstatus", "shows playback status"),
         ("--play", "plays the song"),
         ("--pause", "pauses the song"),
         ("--playpause", "plays or pauses the song (toggles a state)"),
+        ("--lyrics", "shows the lyrics for the song"),
         ("--next", "plays the next song"),
         ("--prev", "plays the previous song"),
         ("--volumeup", "increases the sound volume"),
-        ("--volumedown", "decreases the sound volume")
+        ("--volumedown", "decreases the sound volume"),
```
Comment (pwittchen) [inline spotifycli/spotifycli.py]: Is this comma necessary?

**Comment by pwittchen** [comment]
Thanks! We're almost there. I have only last small issues with the code. Please, have a look at them. Of course, I'll test it before merge ;-).

**Comment by Arsukeey** [comment]
I fixed the issues.

**Comment by pwittchen** [comment]
When I execute:

```
./spotifycli/spotifycli.py 
```

I constantly get:

```
Traceback (most recent call last):
  File "./spotifycli/spotifycli.py", line 9, in <module>
    import lyricwikia
ImportError: No module named lyricwikia
```

but I've already installed lyricwikia with pip :(. Maybe something is wrong with my configuration or it work differently in a scripts like that, but I don't know how to fix it.

**Comment by Arsukeey** [comment]
Have you tried to import it in the terminal? Try to `pip uninstall lyricwikia` and then remove the leftovers from your ~/.local/lib and /usr/local/lib. Then reinstall it and make sure you're installing for 2.7

**Comment by pwittchen** [comment]
Ok, after reinstalling package, it worked :-D.
PR is merged. Moreover, I've removed empty line and added exception handling when lyrics are not found.

**Pull request merged**